File: Client_Root/Client/Assets/Scripts/Network/Messages/ToClient/SelectNormalGameToC.cs
using FlatBuffers;

public class SelectNormalGameToC : IMessage
{
    public const ushort MESSAGE_ID = MessageID.SelectNormalGameToC_ID;

    public int m_nResult;
    public int m_nExpectedTime;

    public override ushort GetID()
    {
        return MESSAGE_ID;
    }

    public override IMessage Clone()
    {
        return null; 
    }

    public override byte[] Serialize()
    {
        return null;
    }

    public override bool Deserialize(byte[] bytes)
    {
        var buf = new ByteBuffer(bytes);

        var data = FBS.SelectNormalGameToC.GetRootAsSelectNormalGameToC(buf);

        m_nResult = data.Result;
        m_nExpectedTime = data.ExpectedTime;

        return true;
    }
}

File: Server_Root/Server/Server/Messages/ToClient/SelectNormalGameToC.cpp
#include "stdafx.h"
#include "SelectNormalGameToC.h"
#include "../../../FBSFiles/SelectNormalGameToC_generated.h"

SelectNormalGameToC::SelectNormalGameToC()
{
}

SelectNormalGameToC::~SelectNormalGameToC()
{
}

unsigned short SelectNormalGameToC::GetID()
{
	return MESSAGE_ID;
}

IMessage* SelectNormalGameToC::Clone()
{
	return NULL;
}

const char* SelectNormalGameToC::Serialize(int* pLength)
{
	FBS::SelectNormalGameToCBuilder data_builder(m_Builder);
	data_builder.add_Result(m_nResult);
	data_builder.add_ExpectedTime(m_nExpectedTime);
	auto data = data_builder.Finish();

	m_Builder.Finish(data);

	*pLength = m_Builder.GetSize();

	return (char*)m_Builder.GetBufferPointer();
}

bool SelectNormalGameToC::Deserialize(const char* pChar)
{
	auto data = flatbuffers::GetRoot<FBS::SelectNormalGameToC>((const void*)pChar);

	m_nResult = data->Result();
	m_nExpectedTime = data->ExpectedTime();

	return true;
}